dist/lang/missing/ko.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should these entries be blank? Does Foundry's localization mechanism fall back to English?

Reply:
These files were present, even in the original project, but that's a good question, I don't think it does the fallback to the English language I would just remove them.